plugins/SitesManager/templates/_wordpressTabInstructions.twig

```diff
@@ -1,33 +1,39 @@
-<p>{{ 'SitesManager_SiteWithoutDataWordpressIntroLine1'|translate }}</p>
-<p>{{ 'SitesManager_SiteWithoutDataWordpressIntroLine2'|translate('<a target="_blank" rel="noreferrer noopener" href="https://matomo.org/faq/new-to-piwik/how-do-i-manually-insert-the-matomo-tracking-code-on-wordpress/">', '</a>', '<a target="_blank" rel="noreferrer noopener" href="https://matomo.org/faq/new-to-piwik/start-tracking-with-a-wordpress-com-site-that-cannot-use-plugins/">', '</a>')|raw }}</p>
+<script type="text/javascript" charset="utf-8">
+    $(document).on( 'click', '#wordpress-email-instructions', function() {
+        var emailTrackingCodeLink = $('.emailTrackingCode');
+        if (emailTrackingCodeLink.length) {
+            emailTrackingCodeLink[0].click();
+        }
+    });
```

Comment:
Sure ,that this link should send the default tracking code integration description? It effeminately sound more like it should send the wordpress ones.

Reply:
I will confirm this with PO and update you

Reply:
@sgiehl "And “click here to send this instructions” is fine to imitate “email instructions”. "